exasol_script_languages_container_tool/lib/tasks/test/test_runner_db_test_task.py

```diff
@@ -56,43 +56,42 @@ def run_task(self):
         export_info = export_infos[self.release_goal]
         self.test_environment_info = self.get_values_from_future(
             self._test_environment_info_future)  # type: EnvironmentInfo
-        reuse_release_container = \
-            self.reuse_database and \
-            self.reuse_uploaded_container and \
-            not export_info.is_new
         database_credentials = self.get_database_credentials()
         yield from self.upload_container(database_credentials,
-                                         export_info,
-                                         reuse_release_container)
-        reuse_databse_setup = not (self.reuse_database_setup and self.test_environment_info.database_info.reused)
-        if reuse_databse_setup:
-            yield from self.populate_test_engine_data(self.test_environment_info, database_credentials)
+                                         export_info)
+        yield from self.populate_test_engine_data(self.test_environment_info, database_credentials)
         test_results = yield from self.run_test(self.test_environment_info, export_info)
         self.return_object(test_results)
 
-    def upload_container(self, database_credentials, export_info, reuse_release_container):
+    def upload_container(self, database_credentials: DatabaseCredentials, export_info: ExportInfo):
+        reuse = \
+            self.reuse_database and \
+            self.reuse_uploaded_container and \
+            not export_info.is_new
         upload_task = self.create_child_task_with_common_params(
             UploadExportedContainer,
             export_info=export_info,
             environment_name=self.test_environment_info.name,
             test_environment_info=self.test_environment_info,
             release_name=export_info.name,
-            reuse_uploaded=reuse_release_container,
+            reuse_uploaded=reuse,
             bucketfs_write_password=database_credentials.bucketfs_write_password
         )
         yield from self.run_dependencies(upload_task)
 
     def populate_test_engine_data(self, test_environment_info: EnvironmentInfo,
                                   database_credentials: DatabaseCredentials) -> None:
-        task = self.create_child_task(
-            PopulateTestEngine,
-            test_environment_info=test_environment_info,
-            environment_name=self.test_environment_info.name,
-            db_user=database_credentials.db_user,
-            db_password=database_credentials.db_password,
-            bucketfs_write_password=database_credentials.bucketfs_write_password
-        )
-        yield from self.run_dependencies(task)
+        reuse = not (self.reuse_database_setup and self.test_environment_info.database_info.reused)
```

Comment:
I still have the feeling that the name `reuse` is wrong as the Assigned boolean condition is wrapped in a `not()`.
Should the task `PopulateTestEngine` be created only if we can **not** reuse the database?
So the logic of the implementation is correct, but the naming is negated and hence contradicting the implementation.

Proposal: 
```python
reuse = self.reuse_database_setup and self.test_environment_info.database_info.reused
if not reuse:
```